modules/oe_content_project/config/install/core.entity_form_display.node.oe_project.default.yml

```diff
@@ -115,7 +115,7 @@ content:
       size: 60
       placeholder: ''
```

Comment:
These seem to be autocomplete widget settings, not inline entity form ones, did you re-export this config after changing it from UI or just replaced entity_reference_revisions_autocomplete with inline_entity_form_simple in here manually?